lib/engine/game/base.rb

```diff
@@ -644,13 +675,11 @@ def process_action(action)
         end
 
         @last_processed_action = action.id
-        self
```

Comment:
why are these removed?

Reply:
they were originally used for the undo functionality, since this isn't the process action it makes no sense